package lectureSeven.exceptions;

import java.io.*;

public class Exercises {
    public static void main(String[] args) {

    }
        /* static void testCheckedException() {
            File file = new File("src/lectureSeven/textFile.txt");
            FileReader fr = new FileReader(file);

        }*/

        /* static void testUncheckedException() {
           int[] num = {1, 2, 3, 4};
            System.out.println(num[5]);
        }*/

        /*static void testTryCatch(String path) {
            File file = new File(path);
            FileReader fileReader;

            try {
                fileReader = new FileReader(file);

            }catch (FileNotFoundException e) {
                System.out.println("The file is not found! Please check the file name and try again");
            }

        }*/

        /*static void testTryMultipleCatch(int number, int factor) {
            try {
                if (number % factor == 0)
                    System.out.println(factor + "is a factor of " + number);
            } catch(ArithmeticException ex) {
                System.out.println("Aritmetic " + ex);
            } catch (NumberFormatException ex) {
                System.out.println("Number Format Exception " + ex);
            }

        }*/


    }

